### PR TITLE
Tof sensor

### PR DESCRIPTION
**Pull request by Soham-West**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -16,7 +16,7 @@
  */
 public final class Constants {
   public static final boolean DRIVETRAIN_DEBUG = false;
-  public static final boolean ARM_DEBUG = false;
+  public static final boolean ARM_DEBUG = true;
```
Comment (bengold22) [inline src/main/java/frc/robot/Constants.java]: Should we commit this as false then enable as needed?

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -137,7 +132,8 @@ private void configureBindings() {
     controller.povDownRight().onTrue(new SetGridSlot(pieceGrid, ArmPositionGrid.DOWN_RIGHT));
     LeftGyroButton.onTrue(new GyroOffseter(drivetrain, +5));
     RightGyroButton.onTrue(new GyroOffseter(drivetrain, -5));
-    controller.button(XboxController.Button.kA.value).onTrue(new MoveArmToGridPosition(arm,extender,pieceGrid));
+    //controller.button(XboxController.Button.kA.value).onTrue(new MoveArmToGridPosition(arm,extender,pieceGrid));
+    controller.button(XboxController.Button.kA.value).onTrue(new AutoCloseGripper(arm, gripper)); //temp
```
Comment (bengold22) [inline src/main/java/frc/robot/RobotContainer.java]: You need to put MoveArmToGridPosition back before merging

File: src/main/java/frc/robot/subsystems/Arm.java [hunk 3]
```diff
@@ -23,7 +27,8 @@ public class Arm extends SubsystemBase {
   public double kP, kI, kD, kIz, kFF, kVoltage;
   private boolean pidding;
   private ProtectionMechanism protectionMechanism;
-
+  private Rev2mDistanceSensor distanceSensor;
+  private boolean heightCheck;
```
Comment (bengold22) [inline src/main/java/frc/robot/subsystems/Arm.java]: Maybe call this distanceCheck now?

**Review by bengold22 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (QuokkaLord) [reply src/main/java/frc/robot/Constants.java]: ok

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (QuokkaLord) [reply src/main/java/frc/robot/RobotContainer.java]: ok

File: src/main/java/frc/robot/subsystems/Arm.java (hunk 3 above)
Comment (QuokkaLord) [reply src/main/java/frc/robot/subsystems/Arm.java]: was temp now removed

**Review by bengold22 — APPROVED** [review]

**Pull request merged**